extensions/panache/hibernate-orm-panache/runtime/src/main/java/io/quarkus/hibernate/orm/panache/runtime/PanacheQueryImpl.java

```diff
@@ -243,5 +311,33 @@ private void manageOffsets(int maxResults) {
             options.setFirstRow(null);
         }
         jpaQuery.setMaxResults(maxResults);
+
+        return jpaQuery;
+    }
+
+    private Query innerCreateQuery() {
```

Comment:
I think this method should be called `createQueryNoPaging` or something like that.

Reply:
`createBaseQuery` reads better no ?